### PR TITLE
Wt 47 add localization

### PR DESCRIPTION
**Pull request by k1mka**

### CODE REVIEW

File: assets/transletions/uk.json [hunk 1]
```diff
@@ -4,6 +4,11 @@
   "sing_in" : "Увійти",
   "welcome_back_tony" : "Ласкаво просимо, Тоні!",
   "dont_have_an_account_yet" : "Ще не маєте облікового запису?",
-  "sing_up" : "Реєстрація"
+  "sing_up" : "Реєстрація",
+  "by_signing_in_you_agree_to_our" : "Увійшовши, ви погоджуєтеся з нашими",
+  "terms" : "умовами",
+  "and" : "та",
+  "privacy_policy" : "політикою конфіденційності."
```
Comment (zddarova) [inline assets/transletions/uk.json:11]: сделаешь Умови та Политика с большой?

**Review by zddarova — APPROVED** [review]

**Comment by zddarova** [comment]
@k1mka in this PR add localization to the UI, and recheck pls that all string are translated and uses localization. So there won't be some button that remain English label while all other widgets on ukrainian

File: lib/presentation/widgets/not_have_account_widget.dart [hunk 2]
```diff
@@ -39,3 +42,15 @@ class NotHaveAccount extends StatelessWidget {
     );
   }
 }
+
+//TextSpan(
```
Comment (AndreyKomarichev10) [inline lib/presentation/widgets/not_have_account_widget.dart:43]: @k1mka remove?

File: lib/presentation/widgets/not_have_account_widget.dart [hunk 3]
```diff
@@ -17,14 +20,14 @@ class NotHaveAccount extends StatelessWidget {
...
+            child: Text.rich(
               TextSpan(
-                text: notHaveAccount,
+                text: notHaveAccount.tr(),
                 style: CustomTheme.notHaveAccountProperty,
                 children: [
                   spaceTextSpan,
                   TextSpan(
-                    text: singUp, // TODO: refactored
+                    text: singUp.tr(), // TODO: refactored
```
Comment (AndreyKomarichev10) [inline lib/presentation/widgets/not_have_account_widget.dart]: why to use static constant for this and not use LocaleKeys.sing_in directly in a build? and remove todo

File: lib/presentation/widgets/logo_form.dart [hunk 4]
```diff
@@ -32,18 +30,18 @@ class _MyLogFormWidgetState extends State<LogoForm> {
       key: formKey,
       child: Column(
         children: [
-          const Text(
-            greetings,
+          Text(
+            LocaleKeys.welcome_back_tony.tr(),
```
Comment (AndreyKomarichev10) [inline lib/presentation/widgets/logo_form.dart:34]: @k1mka tr() method call is not needed everywhere. need to remove it from all places

File: lib/presentation/widgets/privacy_policy_and_terms_widget.dart [hunk 5]
```diff
@@ -1,42 +1,39 @@
...
               recognizer: TapGestureRecognizer()..onTap = () {},
               // TODO:  code to open / launch terms of service link here
             ),
             spaceTextSpan,
-            const TextSpan(
-              text: and,
-              style: TextStyle(fontSize: 16, color: Colors.black), // todo: move to theme
+            TextSpan(
+              text: LocaleKeys.and.tr(),
+              style: const TextStyle(fontSize: 16, color: Colors.black), // todo: move to theme
```
Comment (AndreyKomarichev10) [inline lib/presentation/widgets/privacy_policy_and_terms_widget.dart:32]: move to theme?

File: pubspec.yaml [hunk 6]
```diff
@@ -31,6 +31,7 @@ dependencies:
     sdk: flutter
   flutter_svg: ^1.1.5
   easy_localization: ^3.0.1
+  localized_rich_text: ^0.0.1
```
Comment (AndreyKomarichev10) [inline pubspec.yaml]: do not recomment you to use this package, need to discuss

**Pull request merged**